internal/cli/command/controlplane/up.go

```
@@ -98,26 +98,12 @@ func runUp(options createOptions) error {
 			}
 		}
 
-		for {
-			if bytes, err := json.MarshalIndent(out, "", "  "); err != nil {
-				log.Errorf("failed to marshal descriptor: %v", err)
-				log.Debugf("descriptor: %#v", out)
-			} else {
-				content = string(bytes)
-				_, _ = fmt.Fprintf(os.Stderr, "The current state of the descriptor:\n\n%s\n", content)
-			}
-
-			var open bool
-			_ = survey.AskOne(&survey.Confirm{Message: "Do you want to edit the controlplane descriptor in your text editor?"}, &open, nil)
-			if !open {
-				break
-			}
-
-			_ = survey.AskOne(&survey.Editor{Message: "controlplane descriptor:", Default: content, HideDefault: true, AppendDefault: true}, &content, nil)
-			out = make(map[string]interface{})
-			if err := unmarshal([]byte(content), &out); err != nil {
-				log.Errorf("can't parse descriptor: %v", err)
-			}
+		if bytes, err := json.MarshalIndent(out, "", "  "); err != nil {
+			log.Errorf("failed to marshal descriptor: %v", err)
+			log.Debugf("descriptor: %#v", out)
```

this will fail silently